### PR TITLE
New PieWidget component

### PR DESCRIPTION
**Pull request by builtbyedgar**

### CODE REVIEW

File: src/ui/utils/getChartSerie.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+export default function getChartSerie (chart, index) {
```
Comment (albertoarana) [inline src/ui/utils/getChartSerie.js:2]: This function can be in `chartUtils`

**Review by albertoarana — CHANGES_REQUESTED** [review]

**Review by albertoarana — APPROVED** [review]

**Review by VictorVelarde — CHANGES_REQUESTED** [review]
Nice work, but 3 aspects to review before merging:
1. order legend items desc by % ?
2. default colors
3. possible bug on hover?

See internal discussion at slack

**Comment by VictorVelarde** [comment]
There is a derived issue on histogram and its setColor not working fine, when trying to filter. I'm currently checking it.

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Comment by VictorVelarde** [comment]
Finally merged 🚀  after polishing some details / (cc @borja-munoz)

**Pull request merged**